SBR.md

```diff
@@ -2220,7 +2220,7 @@ n. __Certificate Field:__ `subject:countryName` (OID: 2.5.4.6)
 
 **Note**: 
 
- 1. Legacy Generation profiles MAY omit the `subject:givenName`, `subject:surname`, and `subject:pseudonym` attributes and include only the `subject:commonName` as described in [Section 7.1.4.2.2(a)](#71422-subject-distinguished-name-fields).
+ 1. Legacy Generation profiles MAY omit the `subject:givenName`, `subject:surname`, and `subject:pseudonym` attributes and include only the `subject:commonName` as described in [Section 7.1.4.2.2(a)](#71422-subject-distinguished-name-fields) provided that this latter attribute contains a Personal Name (see Section 3.1.2).
```

Comment:
Here is a proposed update in-line with RFC 2119:

```suggestion
 1. Legacy Generation profiles MAY omit the `subject:givenName`, `subject:surname`, and `subject:pseudonym` attributes and include only the `subject:commonName` as described in [Section 7.1.4.2.2(a)](#71422-subject-distinguished-name-fields). If the `subject:commonName` is included, it MUST contain a Personal Name (see Section 3.1.2).
```

Reply:
@dzacharo this however is more limiting, and it goes against section [7.1.4.2.2](https://github.com/cabforum/smime/blob/main/SBR.md#71422-subject-distinguished-name-fields) (a).

Your proposed language seems to state 2 different requirements. The first one is that a legacy profile may omit the given/sur/pseudonym fields. The second one is that subject:commonName, if included, MUST contain a Personal Name. 

The proposal as suggested by Adriano keeps the option open to use the Mailbox Address or Pseudonym as commonName, but **only if** subject:givenName and subject:surname (or subject:pseudonym) have been included.

If you want to rephrase, I'd suggest updating your suggestion to:
```suggestion
 1. Legacy Generation profiles MAY omit the `subject:givenName`, `subject:surname`, and `subject:pseudonym` attributes and include only the `subject:commonName` as described in [Section 7.1.4.2.2(a)](#71422-subject-distinguished-name-fields). If all of these fields are omitted and the `subject:commonName` is included, it MUST contain a Personal Name (see Section 3.1.2).
```

Reply:
How about:

```suggestion
 1. Legacy Generation profiles MAY omit the `subject:givenName`, `subject:surname`, and `subject:pseudonym` attributes and include only the `subject:commonName` as described in [Section 7.1.4.2.2(a)](#71422-subject-distinguished-name-fields). If the `subject:givenName`, `subject:surname`, and `subject:pseudonym` attributes are omitted and the `subject:commonName` is included, its value MUST be a Personal Name (see Section 3.1.2).
```

Reply:
WFM!